### PR TITLE
Horizontal bar chart - Component tests

### PR DESCRIPTION
**Pull request by v-baambati**

### CODE REVIEW

**Comment by fabricteam** [comment]
## 📊 Bundle size report

<sub>🤖 This report was generated against <a href='https://github.com/microsoft/fluentui/commit/3ef92a485889eee5010ed32b8fc1ec1d8127cc44'>3ef92a485889eee5010ed32b8fc1ec1d8127cc44</a></sub>
<div id="bundleSizeComment9423"/>

**Comment by fabricteam** [comment]
## 🕵 fluentuiv8 No visual regressions between this PR and main
<div id="vrtCommentfluent-ui_VRT_Pipeline_v8"/>

File: packages/react-charting/src/components/HorizontalBarChart/HorizontalBarChartRTL.test.tsx [hunk 1]
```diff
@@ -0,0 +1,338 @@
...
+    container => {
+      // Assert
+      expect(getByClass(container, /chartTitleRight/i)).toHaveLength(3);
+    },
+  );
+
+  testWithWait(
+    'Should render the bars right side value with franctional value when chartDataMode is fraction',
+    HorizontalBarChart,
+    { data: chartPoints, chartDataMode: 'fraction' },
```
Comment (AtishayMsft) [inline packages/react-charting/src/components/HorizontalBarChart/HorizontalBarChartRTL.test.tsx:162]: Adda a test with chartDataMode= {'percentage'}

**Review by AtishayMsft — APPROVED** [review]

**Pull request merged**